modules/mobile/Config.cpp

```diff
@@ -60,6 +61,8 @@ Config::setConfigurationMap( const QVariantMap& cfgMap )
 
     m_cmdInternalStoragePrepare = getString( cfgMap, "cmdInternalStoragePrepare", "ondev-internal-storage-prepare" );
     m_cmdPasswd = getString( cfgMap, "cmdPasswd", "passwd" );
+    m_cmdUsermod = getString( cfgMap, "cmdUsermod", "xargs -I{} -n1 usermod -m -d /home/{} -l {} user");
```

Comment:
```suggestion
    m_cmdUsermod = getString( cfgMap, "cmdUsermod", "xargs -I{} -n1 usermod -m -d /home/{} -l {} **user**");
```

The username `user` is hard coded here. No matter what the distribution sets it's default to this command will result in an attempt to change the name of the user `user` to the user-provided value, not (for example) the `mobian` user.

Reply:
Cannot `cmdUsermod` be altered by the distribution in addition to the `username`?  This is why I added the comment: https://github.com/calamares/calamares-extensions/pull/26/files#diff-ea3aa46160959f51d6f78ebfff3cc04245c82532eb3f57927b81db718cf4db2cR29

Reply:
```suggestion
    m_cmdUsermod = getString( cfgMap, "cmdUsermod", "xargs -I{} -n1 usermod -m -d /home/{} -l {} -c {} user");
```

The `--comment` (a la `chfn`) option will also change the user's full-name to match, which I can push a change for if desired.  Otherwise, IIUC, it will become w/e `username` config is, even if the user changes the username.

Reply:
> Cannot `cmdUsermod` be altered by the distribution in addition to the `username`? This is why I added the comment: https://github.com/calamares/calamares-extensions/pull/26/files#diff-ea3aa46160959f51d6f78ebfff3cc04245c82532eb3f57927b81db718cf4db2cR29

To clarify, in case it's not: `cmdUsermod` is dependent on `username` config, and if a distribution changes `username`, it will need to update the `cmdUsermod` config to match; yes, this works ("`cmdUsermod` **can** be altered by the distribution ...").